src/main/asciidoc/scanner.adoc

```diff
@@ -121,6 +122,7 @@ Describes a database view.
 |====
 | Name | Description
 | name | The name of the view.
+| remarks | The remarks of the view.
```

Comment:
Could we use _comment_ as name instead of _remarks_? IMHO it is closer to the SQL standard. In SQL we also use the term _comment_.

Reply:
It's the term used by `liquibase` and `schemacrawler`, that's why I used it.
But `comment` is fine, that what e.g. `jooq` uses too.
Renamed.